plugins/serializers/prometheusremotewrite/prometheusremotewrite_test.go

```diff
@@ -210,7 +210,7 @@ func TestRemoteWriteSerializeNegative(t *testing.T) {
 		t.Helper()
 		require.NoError(t, err)
 
-		lastMsg := clog.LastError()
+		lastMsg := clog.LastWarning()
```

Comment:
How about
```suggestion
		warnings := clog.Warnings()
		require.NotEmpty(t, warnings)
		lastMsg := warnings[len(warnings)-1]
```
instead of implementing a whole new function? Please note `LastError()` is only there for backward compatibility to not be forced to change existing code...